usaspending_api/database_scripts/etl/award_delta_view.sql

```diff
@@ -92,6 +95,80 @@ LEFT JOIN (
   FROM
     recipient_lookup AS rlv
 ) recipient_lookup ON (recipient_lookup.duns = vw_award_search.recipient_unique_id AND vw_award_search.recipient_unique_id IS NOT NULL)
+LEFT JOIN (
+    WITH closed_periods AS (
+	    -- This is NOT right, since period dates are not the same as submission window close dates,
+	    -- but an approximation for illustration purposes until we have the reporting dates table up
+	    SELECT DISTINCT concat(p.reporting_fiscal_year::text,
+	        lpad(p.reporting_fiscal_period::text, 2, '0')) AS fyp,
+	        p.reporting_fiscal_year, p.reporting_fiscal_period
+	    FROM submission_attributes p
+	    WHERE
+	    -- For COVID, look only after a certain date (likely on/after 2020-04-01, or the start of FY2020 FYP07)
+	        p.reporting_period_start >= '2020-04-01' -- using older date to see data surface
+	        AND now()::date > p.reporting_period_end -- change "period end" with "window close"
+	    ORDER BY p.reporting_fiscal_year DESC, p.reporting_fiscal_period DESC
+    ),
+    eligible_submissions AS (
+        SELECT *
+        FROM submission_attributes s
+        WHERE concat(s.reporting_fiscal_year::text, lpad(s.reporting_fiscal_period::text, 2, '0')) IN (
+            SELECT fyp FROM closed_periods
+        )
+    ),
+    eligible_file_c_records AS (
+        SELECT
+            faba.award_id,
+            CASE WHEN defc.group_name = ('covid_19') THEN faba.gross_outlay_amount_by_award_cpe else 0 END AS gross_outlay_amount_by_award_cpe,
+            CASE WHEN defc.group_name = ('covid_19') THEN faba.transaction_obligated_amount else 0 END AS transaction_obligated_amount,
+            faba.disaster_emergency_fund_code,
+            s.reporting_fiscal_year,
+            s.reporting_fiscal_period
+        FROM financial_accounts_by_awards faba
+        INNER JOIN eligible_submissions s ON s.submission_id = faba.submission_id
+        INNER JOIN disaster_emergency_fund_code defc on defc.code = faba.disaster_emergency_fund_code
```

Comment:
I think I see why you had to do the case statement here. This string of CTEs is attaching ALL DEFCs to the award record as an array, not just COVID-ones. Hence the `ARRAY_AGG(DISTINCT faba.disaster_emergency_fund_code) AS defc` later on. 

So you need to be inclusive of non-covid-defc rows to capture other DEFCs ... but exclude summing those non-covid DEFC's obs/outs in the total_**covid** values. 

That the case? If so, disregard my previous comment, this works.

----

**_Still ..._**
I wonder if ~a `UNION` might be more efficient here. e.g.~ going after totals separate from DEFCs may be more efficient here, esp. since we're already joining on `financial_accounts_by_awards`:

1) Keep the inner join on `defc.group_name = 'covid_19'`, and this will provide one dijoint subset of file C rows that ARE covid-related, from which you can sum. Have this string of CTEs and query _just_ be about getting those covid totals. Don't bother trying to project the DEF codes here.
2) Then, to get the DEFCs associated with each award, do exactly what you're doing in the `transaction_delta_view` below. That is, end this VIEW with:
```sql
LEFT JOIN (
  SELECT
    faba.award_id,
    ARRAY_AGG(
      DISTINCT CONCAT(
        -- ...
       )
     ) tas_paths,
     ARRAY_AGG(
      DISTINCT CONCAT(
        -- ...
       )
     ) tas_components,
    ARRAY_AGG(faba.disaster_emergency_fund_code) defc
 FROM
   treasury_appropriation_account taa
   INNER JOIN financial_accounts_by_awards faba ON (taa.treasury_account_identifier = faba.treasury_account_id)
   INNER JOIN federal_account fa ON (taa.federal_account_id = fa.id)
   INNER JOIN toptier_agency agency ON (fa.parent_toptier_agency_id = agency.toptier_agency_id)
 WHERE
   faba.award_id IS NOT NULL
 GROUP BY
   faba.award_id
) TREASURY_ACCT ON (TREASURY_ACCT.award_id = vw_award_search.award_id);
```

Would that work ^^ ?? Can you try and experiment with performance?